airtime_mvc/application/forms/EmbeddablePlayer.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+class Application_Form_EmbeddablePlayer extends Zend_Form_SubForm
+{
+    public function init()
+    {
+        $this->setDecorators(array(
+            array('ViewScript', array('viewScript' => 'form/embeddableplayer.phtml'))
+        ));
+
+        $displayTrackMetadata = new Zend_Form_Element_Checkbox('player_display_track_metadata');
+        $displayTrackMetadata->setValue(true);
+        $displayTrackMetadata->setLabel(_('Display track metadata?'));
+        $this->addElement($displayTrackMetadata);
+
+        $streamMode = new Zend_Form_Element_Radio('player_stream_mode');
+        $streamMode->setLabel(_('Select Stream:'));
+        $streamMode->setMultiOptions(
+            array(
+                "auto" => "Auto detect the most appropriate stream to use.",
+                "manual" => "Select a stream:"
+            )
+        );
+        $streamMode->setValue("auto");
+        $this->addElement($streamMode);
+
+        $streamURL = new Zend_Form_Element_Radio('player_stream_url');
+        $opusStreamCount = 0;
+        $urlOptions = Array();
+        foreach(Application_Model_StreamSetting::getEnabledStreamData() as $stream => $data) {
+            $urlOptions[$stream] = strtoupper($data["codec"])." - ".$data["bitrate"]."kbps";
+            if ($data["mobile"]) {
+                $urlOptions[$stream] .= " - Mobile friendly";
+            }
+            if ($data["codec"] == "opus") {
+                $opusStreamCount += 1;
+                $urlOptions[$stream] .=" - The player does not support Opus streams.";
+            }
+        }
+        $streamURL->setMultiOptions(
+            $urlOptions
+        );
+
+        foreach ($urlOptions as $o => $v) {
+            if (strpos(strtolower($v), "opus") !== false) {
+                continue;
+            } else {
+                $streamURL->setValue($o);
+                break;
+            }
+        }
+
+        $streamURL->setAttrib('numberOfEnabledStreams', sizeof($urlOptions)-$opusStreamCount);
+        $streamURL->setAttrib("disabled", "disabled");
+        $this->addElement($streamURL);
```

Comment:
The player customization options are hidden (including the embed code) and an error message is displayed stating at least one stream needs to be enabled to use the player.